### PR TITLE
Docs Refactoring

### PR DESCRIPTION
**Pull request by AnastasiaKazantaeva**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: docs/install_guides/uninstalling_openvino.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Uninstalling the Intel® Distribution of OpenVINO™ Toolkit
```
[the remark this answers is not in the document]
Comment (ncybulsk) [reply docs/install_guides/uninstalling_openvino.md:1]: Created one bug: https://jira.devtools.intel.com/browse/CVS-68339

File: docs/install_guides/uninstalling_openvino.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## <a name="uninstall"></a>Uninstall the Toolkit in Linux
+Choose one of the options provided below to uninstall the Intel® Distribution of OpenVINO™ Toolkit from your system.
+
+### Uninstall with GUI
+1. Run the uninstallation script from `<INSTALL_DIR>/openvino_toolkit_uninstaller`:
```
Comment (ncybulsk) [inline docs/install_guides/uninstalling_openvino.md:7]: I would add that <INSTALL_DIR> is /opt/intel/openvino_2021 for a default installation

File: docs/install_guides/deployment-manager-tool.md [hunk 3]
```diff
@@ -30,128 +29,203 @@ There are two ways to create a deployment package that includes inference-relate
...
 
+   *	`u. Provide (or change) path to folder with user data` (optional): Path to a directory with user data (IRs, models, datasets, etc.) files and subdirectories required for inference, which will be added to the deployment archive. By default, it's set to `None`, which means you will separately copy the user data to the target system.
 
-### Run Standard CLI Mode
+   *	`t. Change archive name` (optional): Deployment archive name without extension. By default, it is set to `openvino_deployment_package`.
+ 
+4. Once all the parameters are set, type **g** and press **Enter** to generate the package for the selected target devices. To interrupt the generation process and exit the program, type **q** and press **Enter**.
 
-@sphinxdirective
+   The script successfully completes and the deployment package is generated in the output directory specified.
```
Comment (ncybulsk) [inline docs/install_guides/deployment-manager-tool.md:86]: The script successfully completes and the deployment package is generated in the specified output directory.

File: docs/install_guides/deployment-manager-tool.md [hunk 4]
```diff
@@ -1,24 +1,23 @@
 # OpenVINO™ Deployment Manager Guide {#openvino_docs_install_guides_deployment_manager_tool}
 
-The Deployment Manager of Intel® Distribution of OpenVINO™ creates a deployment package by assembling the model, IR files, your application, and associated dependencies into a runtime package for your target device.
+The Deployment Manager is a Python* command-line tool that creates a deployment package by assembling the model, IR files, your application, and associated dependencies into a runtime package for your target device. This tool is delivered within the Intel® Distribution of OpenVINO™ toolkit for Linux\* and Windows\* release packages and is available after installation in the `<INSTALL_DIR>/deployment_tools/tools/deployment_manager` directory.
```
Comment (ncybulsk) [inline docs/install_guides/deployment-manager-tool.md:3]: OpenVINO™ toolkit for Linux\* and Windows\*

And macOS

File: docs/install_guides/deployment-manager-tool.md [hunk 5]
```diff
@@ -30,128 +29,203 @@ There are two ways to create a deployment package that includes inference-relate
...
-![Deployment Manager configuration dialog](../img/configuration_dialog.png)
-   1. The target devices you have selected at the previous step appear on the screen. If you want to change the selection, type **b** and press **Enter** to go back to the previous screen. 
+  
+@sphinxdirective
+
+.. tab:: Linux  
+
+   .. code-block:: sh
+
+      cd <INSTALL_DIR>/deployment_tools/tools/deployment_manager
```
Comment (ncybulsk) [inline docs/install_guides/deployment-manager-tool.md:46]: Add info that <INSTALL_DIR> is /opt/intel/openvino_2021 by default

File: docs/install_guides/uninstalling_openvino.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Uninstalling the Intel® Distribution of OpenVINO™ Toolkit
+
+## <a name="uninstall"></a>Uninstall the Toolkit in Linux
```
Comment (zKulesza) [inline docs/install_guides/uninstalling_openvino.md:3]: Just a suggestion:
To make the uninstallation document more cohesive/connected with the installation document. I would start each uninstallation option with **Option1:**, **Option2:**

**Pull request merged**